.env

```diff
@@ -0,0 +1 @@
+API_BASE_URL=http://103.127.133.26:5000/
```

Comment:
please remove the .env variable

Reply:
I have removed it. just wondering why the env file is uploaded even though there's .env inside gitignore